src/renderer/dx/DxRenderer.cpp

```diff
@@ -618,6 +619,13 @@ try
         }
         case SwapChainMode::ForComposition:
         {
+            if (!_swapChainHandle)
+            {
+                RETURN_IF_FAILED(DCompositionCreateSurfaceHandle(GENERIC_ALL, nullptr, &_swapChainHandle));

```

Comment:
Per our discussion, this will fail to run on Windows 7 because the dcomp.lib dependency (common.build.post.props) will not be resolved below Windows 8.1.

Replace this with LoadLibrary/GetProcAddress against dcomp.dll instead and remove the export lib link from the props file.